index.html

```diff
@@ -18,14 +24,15 @@
         </div>
         <div class="form-field">
           <label for="level" class="form-label">Level*</label>
-          <select name="level" class="form-input">
+          <select id="level" name="level" class="form-input">
             <option value="basic">Basic</option>
             <option value="advanced">Advanced</option>
           </select>
         </div>
         <div class="form-field">
           <label for="hoursPerWeek" class="form-label">Hours per week*</label>
           <input
+            id="HoursPerWeek"
```

Comment:
Adding id attributes to form elements enhances accessibility and makes it easier to manipulate these elements with JavaScript.